### DISCUSSION TITLE
Multiple listing of assets in the deliverable from Step 2

### DISCUSSION THREAD

**Discussion by K8Sudo** [monarc-project/MonarcAppFO#384] [Ideas]
As noted in the MONARC training: If impact adjustments are made not only at the level of the primary assets but also at the level of the secondary assets, these assets are listed more than once.

Here it could be useful if breadcrumbs were displayed in addition to the asset name in the table heading.
Something like: "Audit & Prüfung >> BBB" and "Consulting >> BBB"
Example:

We have the asset BBB (BigBlueButton) inside our dependency tree four times
![monarc_training_error_1 PNG](https://user-images.githubusercontent.com/47662401/135262869-ef035864-eef6-4bb5-ab5a-0e7ace548dec.png)

Because we have made an adjustment here in the impact scales
![monarc_training_error_2](https://user-images.githubusercontent.com/47662401/135263115-eda8a77d-7721-43e6-9e6f-b6c03c7db386.PNG)

the asset will be listed four times in the report from Section 2.
![monarc_training_error_3](https://user-images.githubusercontent.com/47662401/135263260-11447a7e-fe36-4133-a494-446ac593cfa8.PNG)

**Comment by jfrocha** [comment]
Thanks for the bug report. 
Issue was fixed. https://github.com/monarc-project/MonarcAppFO/issues/387
